### PR TITLE
ENH - okhttp upload

### PR DESCRIPTION
**Pull request by rksh**

### CODE REVIEW

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/background/CountingRequestBody.java [hunk 1]
```diff
@@ -0,0 +1,88 @@
+package net.wigle.wigleandroid.background;
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/background/CountingRequestBody.java]: if this is just demo, should it go away before checkin? or is it handy?

Comment (bobzilladev) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/background/CountingRequestBody.java]: nevermind, firefox search didn't find it being used below.

File: wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/background/CountingRequestBody.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+import okhttp3.RequestBody;
+import okio.Buffer;
+import okio.BufferedSink;
+import okio.ForwardingSink;
+import okio.Okio;
+import okio.Sink;
+
+/**
+ * From https://github.com/hongyangAndroid/okhttputils/blob/master/okhttputils/src/main/java/com/zhy/http/okhttp/request/CountingRequestBody.java
+ * - Apache Licensed, here for demo only at this point
```
Comment (bobzilladev) [inline wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/background/CountingRequestBody.java]: Actually used now :)

Comment (rksh) [reply wiglewifiwardriving/src/main/java/net/wigle/wigleandroid/background/CountingRequestBody.java]: updated - good catch

**Pull request merged**